tests/addresses_test.js

```diff
@@ -119,6 +119,54 @@ test("allows creation with just a company", async function (t) {
   t.equal(response.status, 200);
 });
 
+test("correctly creates an international address", async function (t) {
+  const params = {
+    description: "Harry - Office",
+    name: "Harry Zhang",
+    email: "harry@lob.com",
+    phone: "5555555555",
+    address_line1: "370 WATER ST",
+    address_line2: "",
+    address_city: "SUMMERSIDE",
+    address_state: "PRINCE EDWARD ISLAND",
+    address_zip: "C1N 1C4",
+    address_country: "CA",
+  };
+
+  let response = await prism
+    .setup({ errors: false })
+    .then((client) =>
+      client.post(resource_endpoint, params, { headers: prism.authHeader })
+    );
+
+  await t.doesNotReject(Promise.resolve(response));
+  t.equal(response.status, 200);
+  t.equal(response.data.address_country, "CANADA");
+});
+
+test("errors on receiving incorrect international address input", async function (t) {
```

Comment:
I don't think this test is testing incorrect international address input. The country code defaults to "US", therefore this test would be treated as an incorrect US address.